x-pack/plugins/banners/public/plugin.tsx

```diff
@@ -23,7 +23,7 @@ export class BannersPlugin implements Plugin<{}, {}, {}, BannerPluginStartDepend
     { chrome, uiSettings, http }: CoreStart,
     { screenshotMode }: BannerPluginStartDependencies
   ) {
-    if (!screenshotMode.isScreenshotMode()) {
+    if (!(screenshotMode?.isScreenshotMode() ?? false)) {
```

Comment:
Not strictly necessary, but I took the opportunity to change the screenshotMode dependency from required to optional, given it felt way more logical that way.